src/main/java/finalcountdown/homerecipesystembackend/service/PrepMethodService.java

```diff
@@ -0,0 +1,80 @@
+package finalcountdown.homerecipesystembackend.service;
+
+import finalcountdown.homerecipesystembackend.model.PrepMethod;
+import finalcountdown.homerecipesystembackend.model.Step;
+import finalcountdown.homerecipesystembackend.repository.PrepMethodRepository;
+import finalcountdown.homerecipesystembackend.repository.StepRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Slf4j
+public class PrepMethodService {
+
+    private final PrepMethodRepository prepMethodRepository;
+    private final StepRepository stepRepository;
+
+    public PrepMethodService(PrepMethodRepository prepMethodRepository, StepRepository stepRepository) {
+        this.prepMethodRepository = prepMethodRepository;
+        this.stepRepository = stepRepository;
+    }
+
+    public void savePrepMethod(PrepMethod prepMethod) {
+        prepMethodRepository.save(prepMethod);
+    }
+
+    public void addStepToPrepMethod(Step step) {
```

Comment:
We need to use something like RecipeIngredientRequest in RecipeService addIngredientToRecipe method.

So for example StepToMethodRequest that contains fields Long stepId and Long prepMethodId. (double check the naming)